src/Workspaces/Remote/ServiceHub/Host/RemoteWorkspace.SolutionCreator.cs

```diff
@@ -402,7 +412,8 @@ private async Task<Project> UpdateProjectInfoAsync(Project project, Checksum inf
                         lazyDocumentsToAdd ??= ImmutableArray.CreateBuilder<DocumentInfo>();
 
                         // we have new document added
-                        var documentInfo = await _assetProvider.CreateDocumentInfoAsync(newDocumentChecksums.Checksum, cancellationToken).ConfigureAwait(false);
+                        var documentInfo = await _assetProvider.CreateDocumentInfoAsync(
+                            project.Id, newDocumentChecksums.Checksum, cancellationToken).ConfigureAwait(false);
```

Comment:
> project.Id [](http://example.com/codeflow?start=27&length=11)

nit: everywhere else seems to be name param'ing this

Reply:
can fixup.